frontend/javascripts/oxalis/view/context_menu.tsx

```diff
@@ -1107,6 +1112,46 @@ function ContextMenuInner(propsWithInputRef: Props) {
     maybeViewport,
   } = props;
 
+  const segmentIdAtPosition = globalPosition != null ? getSegmentIdForPosition(globalPosition) : 0;
+  const { visibleSegmentationLayer, volumeTracing } = props;
+  const hasNoFallbackLayer =
+    visibleSegmentationLayer != null &&
+    "fallbackLayer" in visibleSegmentationLayer &&
+    visibleSegmentationLayer.fallbackLayer == null;
+  const [segmentSize, boundingBoxInfo] = useFetch(
+    async () => {
+      if (contextMenuPosition == null) return [];
```

Comment:
not sure why, but if I include this in the checks in the next line, I get the error `Can't perform a React state update on an unmounted component`

Reply:
I'm not sure why, either. probably unrelated to that: you are not returning a tuple here, which could make the unpacking in line 1121 throw an error. seems fishy to me :)

Reply:
when testing it, it didnt throw any errors, but i agree that this should be improved!

Reply:
yes, that's fine :+1: